src/test/integration/blocks.e2e-spec.ts

```diff
@@ -25,11 +25,11 @@ describe('Blocks Service', () => {
 
   describe('Blocks', () => {
     it('blocks should have hash, epoch and shard', () => {
-      for (const block of blocks) {
+      blocks.forEach(block => {
```

Comment:
please use for ... of instead of .forEach

Reply:
fixed